tests/test_more.py

```diff
@@ -2668,7 +2668,20 @@ def test_slicing(self):
         first_slice = mi.islice_extended(iterable)[10:]
         second_slice = mi.islice_extended(first_slice)[:10]
         third_slice = mi.islice_extended(second_slice)[::2]
-        self.assertEqual(list(third_slice), ['10', '12', '14', '16', '18'])
+        self.assertSequenceEqual(list(third_slice),
+                                 ['10', '12', '14', '16', '18'])
+
+    def test_slicing_extensive(self):
+        iterable = range(10)
+        options = (None, 1, 2, 7, -1)
+        for start, stop, step in product(options, options, options):
```

Comment:
Could you use `self.subTest` here, as the [test above](https://github.com/more-itertools/more-itertools/pull/430/files#diff-eb9b02233f873d0b01c09f7c2a396636R2657) does?